lpddr: NameError: name 'mr1' is not defined

```
(LX P=mimasv2 C=vexriscv R=master-clean) tansell@tansell:~/github/timvideos/litex-buildenv-clean$ make firmware                                                                                                   
mkdir -p build/mimasv2_base_vexriscv/
time python -u ./make.py --platform=mimasv2 --target=base --cpu-type=vexriscv --iprange=192.168.100 -Ob toolchain_path /opt/Xilinx/    --no-compile-gateware \                                                    
        2>&1 | tee -a /usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/build/mimasv2_base_vexriscv//output.20191030-132700.log; (exit ${PIPESTATUS[0]})                                       
Traceback (most recent call last):
  File "./make.py", line 164, in <module>
    main()
  File "./make.py", line 148, in main
    vns = builder.build(**dict(args.build_option))
  File "/usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/third_party/litex/litex/soc/integration/builder.py", line 167, in build                                                              
    self._generate_includes()
  File "/usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/third_party/litex/litex/soc/integration/builder.py", line 129, in _generate_includes                                                 
    self.soc.sdram.controller.settings.timing))
  File "/usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/third_party/litedram/litedram/init.py", line 434, in get_sdram_phy_c_header                                                          
    init_sequence, mr1 = get_sdram_phy_init_sequence(phy_settings, timing_settings)
  File "/usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/third_party/litedram/litedram/init.py", line 377, in get_sdram_phy_init_sequence                                                     
    }[phy_settings.memtype](phy_settings, timing_settings)
  File "/usr/local/google/home/tansell/github/timvideos/litex-buildenv-clean/third_party/litedram/litedram/init.py", line 78, in get_lpddr_phy_init_sequence                                                      
    return init_sequence, mr1
NameError: name 'mr1' is not defined

real    0m0.371s
user    0m0.311s
sys     0m0.066s
make: *** [Makefile:303: firmware-cmd] Error 1
```

--- a/litedram/init.py
+++ b/litedram/init.py
@@ -75,7 +75,7 @@ def get_lpddr_phy_init_sequence(phy_settings, timing_settings):
         ("Load Mode Register / CL={0:d}, BL={1:d}".format(cl, bl), mr, 0, cmds["MODE_REGISTER"], 200)
     ]
 
-    return init_sequence, mr1
+    return init_sequence, None
 
 # DDR2 ---------------------------------------------------------------------------------------------
 def get_ddr2_phy_init_sequence(phy_settings, timing_settings):

